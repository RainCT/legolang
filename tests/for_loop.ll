for i in 1 .. 5:
    print i

print ""
for i in 2 .. 4:
    print i

print ""
for i in 10 .. 7:
    print i

print ""
for i in 4.6 .. 5.2:
    print i // this should run just once, with value 5
