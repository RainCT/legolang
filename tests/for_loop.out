1
2
3
4
5

2
3
4

10
9
8
7

5
